arlo_server/all_routes/election/__element__.py

```diff
@@ -0,0 +1,91 @@
+from flask import jsonify, Response, request
+from jsonschema import validate
+
+from arlo_server import app, db
+from arlo_server.auth import require_audit_admin_for_organization
+from arlo_server.models import Election
+
+ELECTION_NAME_KEY = "electionName"
+ELECTION_ONLINE_KEY = "online"
+ELECTION_RANDOM_SEED_KEY = "randomSeed"
+ELECTION_RISK_LIMIT_KEY = "riskLimit"
+ELECTION_STATE_KEY = "state"
```

Comment:
I know that this helps us from making mistakes, but I think it makes the code more confusing to read for me. It's helpful for my mental model to see the JSON in the code.

Do you have a strong reason for pulling out these constants?

Reply:
> Do you have a strong reason for pulling out these constants?

Same as usual, preventing typos. I could buy that the schema validation is enough and that they could be duplicated there.